### PR TITLE
Fix some errors in info.js file

### PR DESCRIPTION
**Pull request by Bryan-0**

### CODE REVIEW

**Comment by TheImmortal** [comment]
uhm, the links are not errors.

File: chat-plugins/info.js [hunk 1]
```diff
@@ -1766,7 +1767,7 @@ var commands = exports.commands = {
 		}
 		if (target === 'all' || target === 'star' || target === 'player') {
 			matched = true;
-			buffer += '<a href="http://www.smogon.com/sim/faq#star">Why is there this star (&starf;) in front of my username?</a><br />';
+			buffer += '<a href=\"http://www.smogon.com/sim/faq#star\">Why is there this star (&starf;) in front of my username?</a><br />';
```
Comment (Rhythms88) [inline chat-plugins/info.js]: Why are you adding forwardslashes on this? It's surrounded in single quotes, so they aren't needed.

**Comment by Bryan-0** [comment]
I know that @TheImmortal, Is just that without "/" can be more ordered.

**Comment by Rhythms88** [comment]
Even if you remove the ending backslashes ( `/` ) on the links, they still get readded once the page is fully loaded.

**Comment by TheImmortal** [comment]
What? "More ordered?" What does that even mean. Next you'll be removing `www` to save space.

File: chat-plugins/info.js [hunk 2]
```diff
@@ -1498,7 +1499,7 @@ var commands = exports.commands = {
 		if (!this.canBroadcast()) return;
 		this.sendReplyBox(
 			"An introduction to the Create-A-Pok&eacute;mon project:<br />" +
-			"- <a href=\"https://www.smogon.com/cap/\">CAP project website and description</a><br />" +
+			"- <a href=\"https://www.smogon.com/cap\">CAP project website and description</a><br />" +
 			"- <a href=\"https://www.smogon.com/forums/showthread.php?t=48782\">What Pok&eacute;mon have been made?</a><br />" +
```
Comment (Rhythms88) [inline chat-plugins/info.js]: The link should change from `https://www.smogon.com/forums/showthread.php?t=48782` to `http://www.smogon.com/forums/threads/48782/`. It still works, even if you don't change it, but it's just to follow the pattern that I've seen.

**Comment by Rhythms88** [comment]
Regardless, these aren't errors, just changes.

File: chat-plugins/info.js (hunk 2 above)
Comment (TheImmortal) [inline chat-plugins/info.js]: Yep, that was the link before Xenforo.

**Comment by TheImmortal** [comment]
Undo those changes. The `/` is part of the link.

**Comment by Bryan-0** [comment]
Ok

**Comment by Bryan-0** [comment]
I'm going to squash the commits.

**Comment by Bryan-0** [comment]
Updated.

File: chat-plugins/info.js [hunk 3]
```diff
@@ -1501,7 +1502,7 @@ var commands = exports.commands = {
 			"- <a href=\"https://www.smogon.com/cap/\">CAP project website and description</a><br />" +
 			"- <a href=\"https://www.smogon.com/forums/showthread.php?t=48782\">What Pok&eacute;mon have been made?</a><br />" +
 			"- <a href=\"https://www.smogon.com/forums/forums/311\">Talk about the metagame here</a><br />" +
-			"- <a href=\"https://www.smogon.com/forums/threads/3512318/#post-5594694\">Sample XY CAP teams</a>"
+			"- <a href=\"https://www.smogon.com/forums/threads/48782/\">Sample XY CAP teams</a>"
```
Comment (TheImmortal) [inline chat-plugins/info.js]: The correct thread is 3512318.

Comment (Bryan-0) [inline chat-plugins/info.js]: Ok.

File: chat-plugins/info.js [hunk 4]
```diff
@@ -1861,23 +1862,23 @@ var commands = exports.commands = {
 		}
 		if (target === 'all' || target === 'bw' || target === 'gen5') {
 			matched = true;
-			buffer += "- <a href=\"https://www.smogon.com/forums/threads/3509218/#post-5522693\">BW Resources</a><br />";
+			buffer += "- <a href=\"https://www.smogon.com/forums/threads/3509218/#post-5522693/\">BW Resources</a><br />";
```
Comment (TheImmortal) [inline chat-plugins/info.js]: `/` is not part of these links...

File: chat-plugins/info.js [hunk 5]
```diff
@@ -98,7 +98,7 @@ var commands = exports.commands = {
 	host: function (target, room, user, connection, cmd) {
 		if (!this.can('rangeban')) return;
 		var self = this;
-		Dnsbl.reverse(target, function(err, hosts) {
+		Dnsbl.reverse(target, function (err, hosts) {
```
Comment (Rhythms88) [inline chat-plugins/info.js]: Zarel already took care of this in https://github.com/Zarel/Pokemon-Showdown/compare/bcd48fe8db5b...2bd4057e15b2

File: chat-plugins/info.js [hunk 6]
```diff
@@ -1816,7 +1817,7 @@ var commands = exports.commands = {
 			matched = true;
 			buffer += "- <a href=\"https://www.smogon.com/tiers/\">Smogon Tiers</a><br />";
 			buffer += "- <a href=\"https://www.smogon.com/forums/threads/tiering-faq.3498332/\">Tiering FAQ</a><br />";
-			buffer += "- <a href=\"https://www.smogon.com/xyhub/tiers\">The banlists for each tier</a><br />";
+			buffer += "- <a href=\"https://www.smogon.com/xyhub/tiers/\">The banlists for each tier</a><br />";
```
Comment (TheImmortal) [inline chat-plugins/info.js]: This is wrong. The link is incorrect with the `/`.

**Comment by Bryan-0** [comment]
Updated. @TheImmortal

**Comment by ascriptmaster** [comment]
Reverse the change that @Rhythms88 mentioned first.

**Comment by Bryan-0** [comment]
Alright, all is done.

File: chat-plugins/info.js [hunk 7]
```diff
@@ -1501,7 +1502,7 @@ var commands = exports.commands = {
 			"- <a href=\"https://www.smogon.com/cap/\">CAP project website and description</a><br />" +
 			"- <a href=\"https://www.smogon.com/forums/showthread.php?t=48782\">What Pok&eacute;mon have been made?</a><br />" +
```
Comment (Rhythms88) [inline chat-plugins/info.js]: you still havent removed the `showthread.php?t=` here

Comment (TheImmortal) [inline chat-plugins/info.js]: and replace it with `threads/`

Comment (Bryan-0) [inline chat-plugins/info.js]: Ohhh, sorry I will do it

Comment (Rhythms88) [inline chat-plugins/info.js]: and add a backslash ( `/` ) after the numbers ( `48782` )

**Comment by Bryan-0** [comment]
Done.

File: chat-plugins/info.js (hunk 1 above)
Comment (Zarel) [inline chat-plugins/info.js]: Technically, those are backslashes.

**Pull request merged**